ADBench/run-all.ps1

```diff
@@ -76,6 +89,12 @@ else {
 assert Test-Path "$dir/ADBench/cmake-vars-$buildtype.ps1"
 . $dir/ADBench/cmake-vars-$buildtype.ps1
 
+# I'm not sure why $gmm_d_vals and $gmm_k_vals are configured using
+# CMake.  That seems rather heavyweight.  Anyway, I added a way to
+# override them here.
```

Comment:
The reason is that cmake has to build separate exectuables for some tools (i.e. the tools use matrices whose size is known at compile time).   So then the tests get told the range of sizes for which exes have been built.   But probably better is just to fail the tests if the exe isn't built.